velox/exec/tests/utils/QueryAssertions.cpp

```diff
@@ -362,6 +362,19 @@ velox::variant arrayVariantAt(
   return velox::variant::array(std::move(array));
 }
 
+namespace {
+velox::variant const getNullDecimalVariant(::duckdb::Value& value) {
+  uint8_t precision;
+  uint8_t scale;
+  value.type().GetDecimalProperties(precision, scale);
+  auto type = DECIMAL(precision, scale);
+  const auto nullValue = (type->isShortDecimal())
+      ? variant::shortDecimal(std::nullopt, type)
```

Comment:
It would be nice to avoid separate logic for handling nulls of decimal type. Can we add support for variant(TypeKind::SHORT_DECIMAL) and variant(TypeKind::LONG_DECIMAL) to allow creating null variants of decimal type?

Reply:
Thanks for the suggestion, currently variant::shortDecimal and variant::longDecimal functions are used to create variants for short and long decimals respectively, so the same functions were used to get Null decimal variants. The variant() constructor is now overloaded to return Null decimal variants.